sdk/communication/azure-communication-rooms/tests/test_rooms_client.py

```diff
@@ -16,17 +17,12 @@
 )
 from azure.communication.rooms._shared.models import CommunicationUserIdentifier
 from _shared.utils import get_http_logging_policy
-from helper import URIIdentityReplacer
-from testcase import RoomsTestCase
+from acs_rooms_test_case import ACSRoomsTestCase
+from devtools_testutils import recorded_by_proxy
 
-class RoomsClientTest(RoomsTestCase):
-    def __init__(self, method_name):
-        super(RoomsClientTest, self).__init__(method_name)
-
-    def setUp(self):
-        super(RoomsClientTest, self).setUp()
-        if not self.is_playback():
-            self.recording_processors.extend([URIIdentityReplacer()])
+class TestRoomsClient(ACSRoomsTestCase):
+    def setup_method(self):
+        super().setUp()
         # create multiple users users
         self.identity_client = CommunicationIdentityClient.from_connection_string(
```

Comment:
It looks like CI is getting upset during this setup, and it makes sense since playback pipeline tests aren't configured for resource deployment. It looks like we'd want to put this under an `if is_live():` check

Reply:
Yup, I've added an `is_live` check and ci pipeline is working for playback tests now :)